gocov/test.go

```diff
@@ -86,12 +90,24 @@ func runTests(args []string) error {
 		// the JSON coverage output.
 		cmd.Stdout = os.Stderr
 		cmd.Stderr = os.Stderr
-		err := cmd.Run()
+
+		commandChan <- cmd
+	}
+
+	for i := 0; i < runtime.NumCPU(); i++ {
```

Comment:
Will the '-p' take effect even for one package?
With current implementation, the '-p=x' will be passed to every package runner, does that mean the explicit goroutine approach is redundant? let me do some local benchmark first.